pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -743,6 +743,20 @@ public enum PGProperty {
       "true",
       "Enable or disable TCP no delay. The default is (@code true}."
   ),
+
+  /**
+   * Connection parameter to control behavior when
+   * {@link Connection#setTransactionIsolation(int)} is set.
+   */
+  TRANSACTION_ISOLATION_MODE(
+      "transactionIsolationMode",
+      "session",
+      "Controls the behavior when a transaction isolation level is set, one of 'transaction', or 'session' "
+          + "When 'transaction' setting isolation level will cause transactions to BEGIN ISOLATION LEVEL if autocommit is 'false'. "
+          + "When 'session' setting readOnly to setting isolation level will set it as the session characteristics.",
+      false,
```

Comment:
I'm unclear what 'setting readOnly to setting isolation level' is trying to convey ?

Reply:
Wrong copy-paste, thanks for catching it